components/list/list-item-expand-collapse-mixin.js

```diff
@@ -127,6 +128,7 @@ export const ListItemExpandCollapseMixin = superclass => class extends superclas
 		}
 		return html`
 			<d2l-button-icon
+				class="${this.skeleton ? 'd2l-skeletize' : ''}"
```

Comment:
I'd recommend using `classMap` here as it should just not output an empty `class` attribute in the case where you don't have a skeleton.